manifests/t/tegan/drakensang/257/tegan.drakensang.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: tegan.drakensang

```

Comment:
The PackageIdentifier is usually in the format of [Publisher].[PackageName]

So should this be called Bigpoint.drakensang